src/angular/directives/dir-datatable.test.js

```diff
@@ -0,0 +1,33 @@
+describe('datatable Directive', function () {
```

Comment:
Since all of the elements using this directive have been replaced with React components, this file is no longer necessary. I'll make a separate commit to remove it.